src/reducers/gcodes.js

```diff
@@ -48,7 +48,7 @@ export default (
         pages = [newPage];
       }
       // This line does everything so far:
-      pages = {data: {items: action.payload.data}, startWith: null};
+      pages = { data: { items: action.payload.data }, startWith: null };
```

Comment:
The spaces inside object literal are the version preferred by linter?

Reply:
Yes, it's ok.